docs/source/conf.py

```diff
@@ -37,6 +37,14 @@
 napoleon_use_param = False
 napoleon_use_ivar = True
 
+# This prevents some errors during the autodoc building process, since
+# errors occur when importing traci and sumolib. A mock import for traci
+# enables some more autodocs to be built, as flow.core.vehicles, but
+# mock imports for sumolib (another module that errors) leads to build failures
+# entirely.
+
+# autodoc_mock_imports = ["traci"]
```

Comment:
why is this commented out?

Reply:
I think that was leftover from a previous build of the docs that was failing for some reason - I'll uncomment it.